db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/UnionNode.java

```diff
@@ -755,14 +755,6 @@ public boolean getIsRecursive() {
         return isRecursive;
     }
 
-    public void setStepNumInExplain(int stepNum) {
-        stepNumInExplain = stepNum;
```

Comment:
Can you remove the variable stepNumInExplain as it is no longer used?

Reply:
Done